### PR TITLE
docs: Update readme

### LINKED ISSUE #58: Missing indices in yaml file
# Describe your question, request or issue here:
I'm using the 0.3.0 release archive.
There are 52 indices in the master table which are missing from the yaml.

<details>
  <summary>Missing indices: (click me!)</summary>
  
- wsdi
- wsdi{ND}
- csdi
- csdi{ND}
- tx95t
- tx{D}tn{D}
- tx{D}tn{D}gt{PRC}p
- txb{D}tnb{D}
- tx{D}tn[D}lt{PRC}p
- gsl
- gsstart
- gsend
- gsgdd
- HI
- BEDD
- HWN(EHF/Tx90/Tn90)
- HWF(EHF/Tx90/Tn90)
- HWD(EHF/Tx90/Tn90)
- HWM(EHF/Tx90/Tn90)
- HWA(EHF/Tx90/Tn90)
- CWN_ECF
- CWF_ECF
- CWD_ECF
- CWM_ECF
- CWA_ECF
- UTCI
- TCI
- TCI60
- TCI80
- r95ptot
- r99ptot
- r{PRC}ptot
- r95p
- r99p
- r75ptot
- r95p
- SPI
- SPEI
- SPI6
- SPI3
- PET
- SSp
- CC
- CC2
- CC6
- CD
- CW
- WD
- WW
- faf
- lsf
- txmax5daymean
</details>

Beside, the index **r75p** is missing from both the xls and the yaml.
I think it's a copy-paste mistake because **r95p** is present twice in rows 126 and 133.

### PR DESCRIPTION
**Pull request by larsbarring**
This improves the README with some general clarification and in particular explains the `ready` column of the spread sheet.

closes #58

### CODE REVIEW

**Comment by larsbarring** [comment]
@bzah: would the suggested changes to the README.md respond to your request in #58 for a better explanation  ?

File: README.md [hunk 1]
```diff
@@ -52,10 +52,66 @@ Considering the very rich and diverse flora of climate indices this
 is however not always possible. By collecting a wide range of
 different indices it is easier to discover any common patterns and
 features that are currently not well covered by the CF Conventions.
-Currently identified issues frequently relate to
-[`standard_name`](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#standard-name)
-or/and [`cell_methods`](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#cell-methods)
-which both are *controlled vocabularies* of the CF Conventions.
+
+With respect to climate index metadata the CF Conventions are in a sense
+both *permissive* and *restrictive*. This have have some bearing on which indices
```
Comment (bzah) [inline README.md:57]:
```suggestion
both *permissive* and *restrictive*. This has some bearing on which indices
```

File: README.md [hunk 2]
```diff
@@ -52,10 +52,66 @@ Considering the very rich and diverse flora of climate indices this
...
+
+In the *index definition* table there is a column *ready* (second left) that is
+subjectively indicating how complete the metadata description is. In general
+terms, indices marked as ready ("1") either have full metadata description,
+or there are advanced plans for what needs to be done. In particular, the focus
+is on the following elements of the CF Conventions:
+
+*  [`standard_name`](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.9/cf-conventions.html#standard-name) (recommended if available).
+
+*  [`long_name`] (free text, recommended).
```
Comment (bzah) [inline README.md:82]: Missing link ? Otherwise the brackets seems unnecessary.
Same goes for unit below.

**Review by bzah — APPROVED** [review]

**Comment by bzah** [comment]
Yes it does @larsbarring, thank you !

**Review by zklaus — APPROVED** [review]

**Pull request merged**